tests/clients/test_securitydata.py

```diff
@@ -13,18 +14,20 @@
 from py42.services.storage.exfiltrateddata import ExfiltratedDataService
 from py42.services.storage.preservationdata import StoragePreservationDataService
 
+py42.settings.use_v2_file_event_data = False
```

Comment:
not needed anymore

Reply:
since `py42.settings` is a module, it's just creating the variable `use_v2_file_event_data` in that namespace.

Reply:
ah right, module not a class